src/components/proposal_list/styles.tsx

```diff
@@ -0,0 +1,162 @@
+import { makeStyles } from '@material-ui/styles';
+
+export const useGetStyles = () => {
+  const useStyles = makeStyles({
+    root: {
+      background: 'white',
+      overflow: 'auto',
+      '& .label': {
+        borderBottom: 'none',
+        fontWeight: 500,
+      },
+      '& .id': {
+        width: '1rem',
+        fontSize: '1rem',
+        verticalAlign: 'top',
+        horizontalAlign: 'left',
+        margin: '1.5rem 0 0 0',
+      },
+      '& .proposer': {
+        display: 'flex',
+        marginRight: '0.5rem',
+      },
+      '& .proposerText': {
+        display: 'none',
+      },
+      '& .mainContent': {
+        display: 'block',
+        margin: '0',
+      },
+      '& .title': {
+        fontWeight: 400,
+      },
+      '& .content': {
+        marginTop: '0.875rem',
+      },
+      '& .time': {
+        marginTop: '0.875rem',
+        display: 'flex',
+        marginBottom: '1rem',
+      },
+      '& .days': {
+        color: '#FD7522',
+      },
+      '& .proposal': {
+        padding: '1.5rem 0 0 0',
+        color: '#414141',
+        display: 'flex',
+        flexDirection: 'column',
+      },
+      '& .button': {
+        width: '100px',
+        height: '30px',
+        color: '#FFFFFF',
+        fontSize: '0.875rem',
+        borderRadius: '6px',
+        border: 'none',
+      },
+      '& .Vote': {
+        background: '#FD3B4C',
+      },
+      '& .Deposit': {
+        background: '#FD7522',
+      },
+      '& .status': {
+        width: '100px',
+        height: '30px',
+        display: 'flex',
+      },
+      '& .statusType': {
+        marginTop: '8px',
+        width: '4px',
+        height: '0.75rem',
+        borderRadius: '8px',
+      },
+      '& .Rejected': {
+        background: '#FD3B4C',
+      },
+      '& .Passed': {
+        background: '#1EC490',
+      },
+      '& .Removed': {
+        background: '#FEA729',
+      },
```

Comment:
add a default grey in .statusType otherwise the styling will break. 

![image](https://user-images.githubusercontent.com/42913823/101852347-66299100-3b98-11eb-9bf9-baed876c7b12.png)